verification/cbmc/include/utils.h

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
```

Comment:
Could you create a GitHub issue to apply this copy style across all AWS Encryption SDK for C? @alex-chew do you agree?

Reply:
That sounds reasonable to me. As part of the issue we would both apply the style to all source code files, and also configure our CI to verify it [1] [2].

[1] https://github.com/awslabs/aws-encryption-sdk-specification/blob/4b1c0e06435d385fc0593a5619ce686cf2cb79e8/.github/workflows/ci_static-analysis.yaml#L7-L12
[2] https://github.com/awslabs/aws-encryption-sdk-specification/blob/master/.github/not-grep.toml

Reply:
Opened it here: #663